CHANGELOG.md

```diff
@@ -15,6 +15,8 @@
     * Update `BTLocalPaymentRequest` to make all properties accessible on the initializer only vs via the dot syntax.
   * BraintreeCard 
     * Remove `BTCardRequest`, use `BTCard` directly instead
+  * BraintreeThreeDSecure
+    * Update `BTThreeDSecureRequest` to make all properties accessible on the initializer only vs via the dot syntax.  
```

Comment:
Let's add an entry as well for any change in types like `BTThreeDSecureRequest.amount` now being a string